src/test/java/com/timekeeper/app/SpringBootTest.java

```diff
@@ -0,0 +1,4 @@
+package com.timekeeper.app;
+
+public @interface SpringBootTest {
+}
```

Comment:
Is this required?  It looks like a result of choosing "Create class" instead of "import" in IntelliJ.